test/depots/depotDonneesHomologations.spec.js

```diff
@@ -1679,4 +1682,93 @@ describe('Le dépôt de données des homologations', () => {
         .catch(done);
     });
   });
+
+  describe('sur une demande de nombre moyen de contributeurs', () => {
+    it("peut retourner le nombre moyen de contributeurs pour les services d'un utilisateur donné", (done) => {
+      const referentiel = Referentiel.creeReferentielVide();
+
+      const proprietaire = new ConstructeurUtilisateur().avecIdUtilisateur(
+        'Propriétaire'
+      ).donnees;
+      const premierContributeur =
+        new ConstructeurUtilisateur().avecIdUtilisateur('A').donnees;
+      const secondContributeur =
+        new ConstructeurUtilisateur().avecIdUtilisateur('B').donnees;
+      const troisiemeContributeur =
+        new ConstructeurUtilisateur().avecIdUtilisateur('B').donnees;
+
+      const premierService = new ConstructeurService()
+        .avecId('123')
+        .avecNomService('un premier service').donnees;
+      const secondService = new ConstructeurService()
+        .avecId('456')
+        .avecNomService('un autre service').donnees;
+
+      const adaptateurPersistance =
+        AdaptateurPersistanceMemoire.nouvelAdaptateur({
+          utilisateurs: [
+            proprietaire,
+            premierContributeur,
+            secondContributeur,
+            troisiemeContributeur,
+          ],
+          homologations: [premierService, secondService],
```

Comment:
Par soucis d'homogénéité avec le reste de la base de code, je pense qu'il faudrait aussi nourrir 

```js
services: [premierService, secondService]
```